src/cljs/pyregence/config.cljs

```diff
@@ -607,12 +607,26 @@
                                                               [:br]
                                                               [:br]
                                                               [:strong "GridFire"]
-                                                              " is a fire behavior model developed by Gary Johnson of Spatial Informatics Group. It combines empirical equations from the wildland fire science literature with the performance of a raster-based spread algorithm using the method of adaptive time steps and fractional distances."]
-                                                 :options    {:elmfire  {:opt-label "ELMFIRE"
-                                                                         :filter    "elmfire"}
-                                                              :gridfire {:enabled?  #(feature-enabled? :gridfire)
-                                                                         :opt-label "GridFire"
-                                                                         :filter    "gridfire"}}}
+                                                              " is a fire behavior model developed by Gary Johnson of Spatial Informatics Group. It combines empirical equations from the wildland fire science literature with the performance of a raster-based spread algorithm using the method of adaptive time steps and fractional distances."
+                                                              [:br]
+                                                              [:br]
+                                                              [:strong "Pyretechnics"]
+                                                              " is a Python"
+                                                              [:a {:href   "https://github.com/pyregence/pyretechnics/ "}
+                                                                " open source"]
+                                                              [:a {:href   "https://pypi.org/project/pyretechnics/"}
+                                                                " library"]
+                                                              ", created by the PyreCast team, that provides modules for surface, crown, and spot fire behavior along with areal burning and fire perimeter tracking. In PyreCast, it is configured to run using a novel implementation of the Eulerian Level Set Model of Fire Spread, similar to the ELMFIRE model. Comprehensive documentation can be found on its official documentation"
+                                                              [:a {:href  "https://pyregence.github.io/pyretechnics/"}
+                                                               " site."]]
```

Comment:
I don't think we need to link to ELMFIRE again since we already link to it above.
```suggestion
                                                              ", created by the PyreCast team, that provides modules for surface, crown, and spot fire behavior along with areal burning and fire perimeter tracking. In PyreCast, it is configured to run using a novel implementation of the Eulerian Level Set Model of Fire Spread, similar to the ELMFIRE model.]
```
What do you think?

Reply:
see dc8997b1646d8fc75b0b3ba0c705784f02677a98